term.c

```diff
@@ -1614,7 +1617,10 @@ term_drain(int fd)
            the port is immediately reconfigured, even after a
            drain. (I guess, drain does not wait for everything to
            actually be transitted on the wire). */
-        if ( DRAIN_DELAY ) usleep(DRAIN_DELAY);
+        if ( DRAIN_DELAY ) {
+            struct timespec d = {0, DRAIN_DELAY * 1000};
+            nanosleep(&d, &s);
```

Comment:
Since we are not checking the return value anyhow, I'd suggest to remove &s and use simply NULL here.

Reply:
Done.